esp/bindings/http/client/httpclient.cpp

```diff
@@ -79,6 +79,7 @@ void CHttpClientContext::initPersistentHandler()
         DBGLOG("Persistent connection won't be enabled for httpclient because maxPersistentIdleTime or maxPersistentRequests is set to 0");
         return;
     }
+    // todo - logging anything here that needs to be done?
     m_persistentHandler.setown(createPersistentHandler(nullptr, maxIdleTime, maxReqs, static_cast<PersistentLogLevel>(getEspLogLevel()), true));
```

Comment:
This reference to esploglevel should be removed, if outside the scope of this PR, please open jira and post here.

Reply:
Either update https://track.hpccsystems.com/browse/HPCC-27215 to include  Persistenthandler, or open a new jira and post here